wheel_radius: 0.1 # robot wheel radius in meters
wheel_base : 0.60 # distance between two wheels in meters

#port: '/dev/OCE_ORRO_DRIVE'
#port: '/dev/oriental'
# port: '/dev/ttyUSB0'
port: '/dev/hb_wheel'
# port_arm: '/dev/ttyUSB1'
port_arm: '/dev/hb_arm'
left_motor_id: 2 #3 
right_motor_id: 3 #2
arm_motor_id: 4 #1
gear_ratio: 100
max_RPM: 3000

imu_number: 0 # roll
